input/pagecontent/comparison.md

```diff
@@ -1,1206 +1,395 @@
-                <p>
-                    <img src="https://hl7.org/fhir/R4/assets/images/tick.png"/> noting that US Core extensibly binds to CVX which is
-                    not an AU vaccine terminology
-                </p>
-            </td>
+            <td style="width: 25%; text-align: center; vertical-align: middle;"><a href="StructureDefinition-au-core-smokingstatus.html">AU Core Smoking Status</a></td>
+            <td style="width: 25%; text-align: center; vertical-align: middle;"><img src="green_checkmark.svg.png" width="20"/></td>
+            <td style="width: 25%; text-align: center; vertical-align: middle;"><img src="cross_red_circle.svg.png" width="20"/></td>
+            <td style="width: 25%; text-align: center; vertical-align: middle;"><img src="green_checkmark.svg.png" width="20"/></td>
         </tr>
         <tr>
-            <td width="93" valign="top">
-                <p>
-                    <a
-                        href="StructureDefinition-au-core-location.html"
-                    >
-                        AU Core Location
-                    </a>
-                </p>
-            </td>
-            <td width="81" valign="top">
-            </td>
-            <td width="111" valign="top">
-            </td>
-            <td width="68" valign="top">
-            </td>
-            <td width="130" valign="top">
-            </td>
-            <td width="78" valign="top">
-                <p>
-                    <a
-                        href="http://hl7.org/fhir/us/core/StructureDefinition/us-core-location"
-                    >
-                        US Core Location
-                    </a>
-                </p>
-            </td>
-            <td width="137" valign="top">
-                <p>
-                    <img src="https://hl7.org/fhir/R4/assets/images/tick.png"/> if Location.name is supplied
-                </p>
-            </td>
+            <td style="width: 25%; text-align: center; vertical-align: middle;"><a href="StructureDefinition-au-core-waistcircum.html">AU Core Waist Circumference</a></td>
+            <td style="width: 25%; text-align: center; vertical-align: middle;"><img src="green_checkmark.svg.png" width="20"/></td>
+            <td style="width: 25%; text-align: center; vertical-align: middle;">&nbsp;</td>
+            <td style="width: 25%; text-align: center; vertical-align: middle;"><img src="green_checkmark.svg.png" width="20"/></td>
         </tr>
+    </tbody>
+</table>
+
+#### International Patient Access 1.0.0
+
+This table provides detailed explanations for instances where AU Core profiles do not comply with the corresponding profiles from International Patient Access.
+
+<table border="1" style="width: 100%; margin: auto; border-collapse: collapse;">
+    <thead>
+        <tr>
+            <th style="width: 33.33%;">AU Core</th>
+            <th style="width: 33.33%;">International Patient Access 1.0.0</th>
+            <th style="width: 33.33%;">Comment</th>
```

Comment:
1. Instead of Comment it should be "Additional Requirements"
2. Add a column for element